README.md

```diff
@@ -790,7 +790,7 @@ There is a `-s` flag for `nvm install` which requests nvm download Node source a
 If installing nvm on Alpine Linux *is* still what you want or need to do, you should be able to achieve this by running the following from you Alpine Linux shell:
 
 ```sh
-apk add -U curl bash ca-certificates openssl ncurses coreutils python2 make gcc g++ libgcc linux-headers grep util-linux binutils findutils
+apk add -U curl bash ca-certificates openssl ncurses coreutils python3 make gcc g++ libgcc linux-headers grep util-linux binutils findutils
```

Comment:
are there older versions of alpine that still work with python2, for users who want to install the majority of older node versions that don't support python 3?

Reply:
Yes, there are.

Reply:
In that case, let's include both sets of instructions.

Reply:
I have done that, at your request, as well as added a side note for Alpine 3.5 - 3.12, regarding older versions of NodeJS.

Reply:
I maintain hundreds of npm packages that support down to node 0.4; EOL dates are, as always, irrelevant.

Reply:
That’s a fair point, haha